tests/phpunit/test-admin-interface.php

```diff
@@ -99,8 +115,10 @@ public function test_humanized_max_age( $max_age, $expected ) {
 	 * @return array
 	 */
 	public function humanized_max_age_provider() {
+		$five_mins = $this->get_five_minutes();
+		var_dump( $five_mins );
```

Comment:
```suggestion
```

Reply:
I caught it in the other PR. Going to just merge so I can just merge both and then make a release PR with the changelog updates and version bump